### ISSUE TITLE
opencv library uses illegal reflection operations in later Java versions

### ISSUE THREAD

**Issue by wayfarer3130** [dcm4che/dcm4che#1077]
In JDK >= 11, the opencv library gets error messages indicating they are using an illegal reflective operation, which will be going away soon as being allowed.

**Comment by nroduit** [comment]
Yes, this is the current integration, but the future implementation with OpenCV does not contain any more reflections and has a simpler and more direct API.

The problem described here has already been [fixed in the dcm4che commands](https://github.com/dcm4che/dcm4che/pull/1034). For integrations in a third-party application, you have to do the same thing.

**Comment by wayfarer3130** [comment]
@nroduit  - That change is at best a work around - you are hiding the illegal reflection.  Having a clean future implementation without reflections would be good, but in the meantime it is cleaner to have an integration that doesn't rely on ignore warning commands because there are implementations which don't actually use FileImageInputStream or FileCacheImageInputStream.

One way to do that is as this change proposes, to add a method to access the raw underlying data.  That method also can work with other implementations of the image input stream, as long as there actually is a local file somewhere containing the data.
